examples/sandbox/src/App.js

```diff
@@ -167,7 +167,8 @@ const config = {
       size: 4
     }
   },
-  apiConnector: connector
+  apiConnector: connector,
+  a11yNotifications: true
```

Comment:
Good call turning this off in the Sandbox. I think we should turn it on in the Elasticsearch example as well.